testing/test_skipping.py

```diff
@@ -713,6 +714,27 @@ def doskip():
     assert result.ret == 0
 
 
+def test_skipped_folding(testdir):
+    testdir.makepyfile(
```

Comment:
i strongly suspect this could produce incorrect results for individual tests marked with skips while a global non skip pytestmark is applied

unfortunately we currently have no mechanism to identify the scope of a skip marker